core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -46,6 +46,7 @@
     private Object mDefaultValue;
     private String mDescription;
     private final String mName;
+    private boolean mIgnoredSiteProperty;
```

Comment:
nit: I will call this `mIgnoringSiteProperty`

Reply:
I prefer the original `mIgnored` - boolean parameters are usually adjectives, not verbs

Reply:
Done.